hack/update-bazel.sh

```diff
@@ -23,4 +23,14 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 git config http.https://gopkg.in.followRedirects true
 
 go get -u gopkg.in/mikedanese/gazel.v14/gazel
-"${GOPATH}/bin/gazel" -root="$(kube::realpath ${KUBE_ROOT})"
+
+for path in ${GOPATH//:/ }; do
+    if [[ -e $path/bin/gazel ]]; then
+      gazel="$path/bin/gazel"
```

Comment:
maybe `break` after this? otherwise this will find the "last" gazel in your $GOPATH, which is the opposite of normal $PATH behavior.